packages/bot-utils/package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@ethersproject/experimental": "^5.7.0",
-    "@mangrovedao/mangrove.js": "1.4.18-1",
+    "@mangrovedao/mangrove.js": "https://github.com/mangrovedao/mangrove.js.git#fee14425efd0fa83fea21c2a18d4404c08935b6c",
```

Comment:
We should probably use a real mangrove.js version

Reply:
Yes but as soon as you validate the PR I will publish a new one :)